composer.json

```diff
@@ -34,6 +34,9 @@
         ]
     },
     "scripts": {
+        "pre-install-cmd": [
+            "cp .env.example .env"
```

Comment:
the example file is an example file, I don't think it's a good idea to copy automaticaly into the the folder. But a warning echo would be a better solution, I think.